#!/usr/bin/bash
def raise_exception():
    now TypeError
